packages/ui/components/CopyToClipboard.tsx

```diff
@@ -28,19 +29,24 @@ interface CopyToClipboardProps {
     end: number
   }
   loading?: boolean
+  className?: string
 }
 
 export const CopyToClipboard: FC<CopyToClipboardProps> = ({
   value,
   success = 'Copied to clipboard!',
   takeN,
   takeStartEnd,
+  className = 'font-mono text-xs',
 }) => {
   const [copied, setCopied] = useState(false)
 
   return (
     <button
```

Comment:
can we set the `title` on this to the un-concat address?> this way when you hover mouse you'll see full address. what do you think?

Reply:
Great idea!